package.json

```diff
@@ -8,7 +8,90 @@
     "contract",
     "ui"
   ],
+  "devDependencies": {
+    "@agoric/eslint-config": "^0.4.1-u12.0",
+    "@endo/eslint-plugin": "^2.0.0",
+    "@jessie.js/eslint-plugin": "^0.4.0",
+    "@typescript-eslint/eslint-plugin": "^6.1.0",
+    "ava": "^5.3.0",
+    "eslint": "^8.47.0",
+    "eslint-config-airbnb-base": "^15.0.0",
+    "eslint-config-jessie": "^0.0.6",
+    "eslint-config-prettier": "^9.1.0",
+    "eslint-plugin-github": "^4.10.0",
+    "eslint-plugin-import": "^2.25.3",
+    "eslint-plugin-jsdoc": "^46.4.3",
+    "eslint-plugin-prettier": "^5.0.0",
+    "prettier": "^3.0.3",
+    "typescript": "~5.3.2"
+  },
+  "eslintConfig": {
+    "env": {
+      "node": true
+    },
+    "parser": "@typescript-eslint/parser",
+    "parserOptions": {
+      "project": "./tsconfig.json",
+      "sourceType": "module",
+      "ecmaVersion": 2020
+    },
+    "extends": [
+      "plugin:@typescript-eslint/recommended",
+      "@agoric"
+    ],
+    "plugins": [
+      "@typescript-eslint",
+      "prettier"
+    ],
+    "rules": {
+      "@typescript-eslint/prefer-ts-expect-error": "warn",
+      "@typescript-eslint/no-floating-promises": "warn",
+      "no-void": [
+        "error",
+        {
+          "allowAsStatement": true
+        }
+      ],
+      "prettier/prettier": "warn",
+      "@typescript-eslint/no-unused-vars": "off"
+    },
+    "ignorePatterns": [
+      "_agstate",
+      "contract/node_modules",
+      "contract/bundles"
+    ]
+  },
+  "prettier": {
+    "trailingComma": "all",
+    "arrowParens": "avoid",
+    "singleQuote": true
+  },
+  "peerDependencies": {
+    "@typescript-eslint/parser": "^5.55.0"
+  },
   "resolutions": {
+    "ses": "0.18.4",
+    "@endo/bundle-source": "2.5.2-upstream-rollup",
+    "@endo/captp": "3.1.1",
+    "@endo/compartment-mapper": "0.8.4",
+    "@endo/far": "0.2.18",
+    "@endo/init": "0.5.56",
+    "@endo/marshal": "0.8.5",
+    "@endo/nat": "4.1.27",
+    "@endo/promise-kit": "0.2.56",
+    "@endo/lockdown": "0.1.28",
+    "@endo/import-bundle": "0.3.4",
+    "@endo/base64": "0.2.31",
+    "@endo/zip": "0.2.31",
+    "@endo/eventual-send": "0.17.2",
+    "@endo/patterns": "0.2.2",
+    "@endo/stream": "0.3.25",
+    "@endo/exo": "0.2.2",
+    "@endo/pass-style": "0.1.3",
+    "@endo/check-bundle": "0.2.18",
+    "@endo/ses-ava": "0.2.40",
+    "@endo/netstring": "0.3.26",
+    "@endo/stream-node": "0.2.26",
     "@babel/code-frame": "7.18.6",
     "@babel/highlight": "7.22.5"
```

Comment:
FYI you should be able to remove this, and upgrade the babel dependencies to their latest now that a fix has been release upstream.

Reply:
will try that out after merging this version, which is tested